pkg/utils/utils.go

```diff
@@ -81,6 +82,20 @@ func GetPodIdentifier(podName, podNamespace string) string {
 	return podIdentifierPrefix + "-" + podNamespace
 }
 
+func addDots(podName string) string {
```

Comment:
Can't we just use `strings.Replace(podName, ".", "_", -1)`?

Reply:
That's a lot cleaner, yeah we can